tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,315 @@
+
+Godot 4 includes three renderers, or *rendering methods*, which use different
+*rendering drivers*:
+
+- **Forward+**, sometimes called **Clustered Forward**. This renderer uses
+  **Vulkan**, **Direct3D 12**, or **Metal** as the rendering driver. The most advanced
+  graphics backend, suited for desktop platforms only. Used by default on 
+  desktop platforms.
+- **Forward Mobile**, sometimes called **Mobile**. This renderer uses
+  **Vulkan**, **Direct3D 12**, or **Metal** as the rendering driver. Less features, 
+  but renders simple scenes faster. Suited for mobile and desktop platforms.
+  Used by default on mobile platforms.
+- **Compatibility**, sometimes called **GL Compatibility**.  This renderer uses
+  **OpenGL 3.3**, **OpenGL ES 3.0**, or **WebGL 2.0** as the rendering driver. The least
+  advanced graphics backend, suited for low-end desktop and mobile platforms.
+  Used by default on the web platform.
+
+Choosing a renderer
+-------------------
+
+Choosing a renderer is a complex question, and depends on your hardware and the
+which platforms you are developing for. As a starting point:
+
+Choose **Forward+** if:
+
+    - You are developing for desktop.
+    - You have relatively new hardware which supports Vulkan.
+    - You are developing a 3D game.
+    - You want to use the most advanced rendering features.
+
+Choose **Mobile** if:
+
+    - You are developing for newer mobile devices, XR, or desktop.
+    - You have relatively new hardware which supports Vulkan.
+    - You are developing a 3D game.
+    - You want to use advanced rendering features, subject to the limitations
+      of mobile hardware.
+
+Choose **Compatibility** if:
+
+    - You are developing for older mobile devices, or older desktop devices. The
+      Compatibility renderer supports the widest range of hardware.
+    - You have older hardware which does not support Vulkan. In this case,
+      Compatibility is the only choice.
+    - You are developing a 2D game, or a 3D game which does not need advanced
+      rendering features.
+
+Keep in mind every game is unique, and this is only a starting point. For example,
+you might choose to use the Compatibility renderer even though you have the latest
+GPU, so you can support the widest range of hardware. Or you might want to use the
+Forward+ renderer for a 2D game, so you can advanced features like compute shaders.
+
+Switching between renderers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the editor, you can always switch between renderers by clicking on the renderer
+name in the upper-right corner of the editor.
+
+Switching between renderers may require some manual tweaks to your scene, lighting,
+and environment, since each renderer is different. In general, switching between
+the Forward+ or Mobile renderers and the Compatibility renderer will require more
+adjustments than switching between the Mobile and Forward+ renderers.
+
+Switching between different renderers at runtime is not yet supported.
```

Comment:
```suggestion
Since Godot 4.4, when using Forward+ or Mobile, if Vulkan is not supported, the engine will fall back to Direct3D 12 and vice versa. If the attempted fallback driver is not supported either, the engine will then fall back to Compatibility when RenderingDevice backend is not supported. This behavior can be disabled in the project settings by unchecking **Rendering > Rendering Device > Fallback to OpenGL 3**.

Switching between different renderers at runtime is not yet supported.
```